run-tests.php

```diff
@@ -57,6 +57,11 @@ function main()
 	// Parallel testing
 	global $workers, $workerID;
 
+	// Minimum required PHP version.
+	if (version_compare(phpversion(), '7.0.0', '<=')) {
+		error('At least PHP 7.0.0 is required to execute run-tests.php.');
+	}
```

Comment:
This doesn't make sense: If you're running an older version of PHP you're going to get a syntax error when executing this script. This message will never be printed.

Reply:
Hm, yes good one. But this applies also if the syntactic change is after the check? Because this check is at the very top of the script and executes first when main() is called...

Reply:
Yes, the syntax error will be through before any code is executed. You can't really do a version check in the same script that contains the new syntax. (Well you can, but it will involve eval...)

Reply:
Ok. Got it... With few dummy tests it worked ok but it really is not executed, yes. Doh... Then I'm adding a simple comment in the script main header which could serve for such notices and syntax things.